app/serializers/motion_serializer.rb

```diff
@@ -18,6 +17,7 @@ class MotionSerializer < ActiveModel::Serializer
 
   has_one :author, serializer: UserSerializer, root: 'users'
   has_one :outcome_author, serializer: UserSerializer, root: 'users'
+  has_one :discussion
```

Comment:
Thank you. Other option is we take your first advice and put group_id in to MotionSerializer. I think we'll do that, but fall back to this if we need discussions in the end.